src/scripts/toDoListsHandler.js

```diff
@@ -0,0 +1,129 @@
+import {
+  postToDoList,
+  getToDoList,
+  deleteRequest,
+  postTask
+} from "./fetchHandler";
+import { toDoList } from "./renderToDoList";
+import { toDoTask } from "./renderToDoTask";
+
+export const toDoListsHandler = async () => {
+  const addNewTaskList = getAddTaskListBtn();
+  addListenerToAddTaskBtn(addNewTaskList);
+  addListenerToTaskList();
+  const apiData = await getToDoList("api/todolists");
+  renderHandler(apiData);
+};
+
+const getAddTaskListBtn = () => {
+  return document.querySelector("#js-add-new-task-list");
+};
+
+const getInputValue = () => {
+  return document.querySelector("#js-new-task-list").value;
+};
+
+const getInput = () => {
+  return document.querySelector("#js-new-task-list");
+};
+
+const getTaskListRenderTarget = () => {
+  return document.querySelector("#js-task-list-target");
+};
+
+const getTaskRenderTarget = taskListId => {
+  return document
+    .querySelector(`[data-id="${taskListId}"]`)
+    .querySelector(".js-task-render-target");
+};
+
+const deleteTaskList = e => {
+  e.target.parentNode.parentNode.remove();
+};
+const deleteTask = e => {
+  e.target.parentNode.remove();
+};
+
+const getCurrentToDoListId = e => {
+  return e.target.parentNode.parentNode.dataset.id;
+};
+
+const getCurrentTaskId = e => {
+  return e.target.parentNode.dataset.id;
+};
+
+const renderList = (apiResponse, element) => {
+  apiResponse.forEach(response => {
+    element.insertAdjacentHTML("afterbegin", toDoList(response));
+  });
+};
+const renderTask = (apiResponse, element) => {
+  apiResponse.forEach(response => {
+    element.insertAdjacentHTML("afterbegin", toDoTask(response));
+  });
+};
+
+const clearList = element => {
+  element.innerHTML = "";
+};
+
+const renderHandler = apiResponse => {
+  const taskListRenderTarget = getTaskListRenderTarget();
+  clearList(taskListRenderTarget);
+  renderList(apiResponse, taskListRenderTarget);
+  for (let i = 0; i < apiResponse.length; i++) {
+    const taskRenderTarget = getTaskRenderTarget(apiResponse[i].id);
+    renderTask(apiResponse[i].tasks, taskRenderTarget);
+  }
+};
+
+const clearInput = element => {
+  element.value = "";
+};
+
+const addTaskBtnHandler = async () => {
+  const inputValue = getInputValue();
+  const input = getInput();
```

Comment:
Jeśli inputa wykorzystujesz dopiero w ifie to nie ma sensu go tu deklarować, jeśli if nie przejdzie to ten kod się zbędnie wykona